Makefile

```diff
@@ -268,3 +272,24 @@ README.md: docs/ ## Generates README.md based on the contents of the docs direct
 
 download_acuant_sdk: ## Downloads the most recent Acuant SDK release from Github
 	@scripts/download_acuant_sdk.sh
+
+clobber_db: ## Resets the database for make setup
+	bin/rake db:create
+	bin/rake db:environment:set
+	bin/rake db:reset
+	bin/rake db:environment:set
+	bin/rake dev:prime
+
+clobber_assets: ## Removes (clobbers) assets
+	bin/rake assets:clobber
+	RAILS_ENV=test bin/rake assets:clobber
```

Comment:
I'd personally find some value in this one, curious if it is aware of all of the asset outputs, i.e. `public/assets` (Sprockets), `public/packs` (Webpack/`jsbundling-rails`), `app/assets/builds` (Sass/`cssbundling-rails`).

Reply:
https://www.rubydoc.info/gems/sprockets-rails/2.3.3 suggests it isn't natively aware of some of those. `public/packs` is still there (15MB), for example. It looks like we can add them in an initializer, but I'd like to propose that be a separate PR if warranted.

Reply:
Yeah, to be fair, that seems like something the `jsbundling-rails` gem should be handling, since it's already doing things with other Rake tasks like `rake assets:precompile`.

Reply:
@n1zyy Based on what I see here, it can be achieved with `javascript:clobber` task:

https://github.com/rails/jsbundling-rails/issues/87#issuecomment-1041234006

Reply:
@aduth I like the idea, but I can't seem to verify what it's actually removing.

https://github.com/rails/jsbundling-rails/blob/main/lib/tasks/jsbundling/clobber.rake suggests that it may automatically extend `assets:clobber`.

Reply:
(More concretely: I can see that it runs `rm_rf Dir["app/assets/builds/**/[^.]*.{js,js.map}"], verbose: false`, but I don't actually _have_ those files when running the app locally and don't follow why.)

Reply:
Oh! I forgot that we do things a little custom such that we don't output JavaScript to `app/assets/builds`, which probably explains why you're not seeing them and not seeing the task have any meaningful impact. We'd probably also need something custom to delete `public/packs`, then.